apps/native-component-list/src/screens/Video/VideoSurfaceTypeScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import { Platform } from 'expo-modules-core';
+import { useVideoPlayer, VideoPlayer, VideoView } from 'expo-video';
+import { View, Text, StyleSheet, ScrollView } from 'react-native';
+
+import { bigBuckBunnySource } from './videoSources';
+import { styles } from './videoStyles';
+
+function ProblematicOverlap({
+  player1,
+  player2,
+  surfaceType,
+  title,
+}: {
+  player1: VideoPlayer;
+  player2: VideoPlayer;
+  surfaceType: 'textureView' | 'surfaceView';
+  title: string;
+}) {
+  return (
+    <View>
+      <Text style={[styles.centerText, styles.mediumText]}>{title}</Text>
+      <View style={[styles.row, { maxWidth: '100%' }]}>
+        <VideoView
+          player={player1}
+          style={[styles.video, borderStyle.border]}
+          surfaceType={surfaceType}
+          contentFit="cover"
+        />
+        <VideoView
+          player={player2}
+          style={[styles.video, borderStyle.border, { left: -150 }]}
+          surfaceType={surfaceType}
+          contentFit="cover"
+        />
+      </View>
+    </View>
+  );
+}
+
+export default function VideoDRMScreen() {
+  const player1 = useVideoPlayer(bigBuckBunnySource, (player) => {
+    player.loop = true;
+    player.play();
+  });
+  const player2 = useVideoPlayer(bigBuckBunnySource, (player) => {
+    player.loop = true;
+    player.play();
+  });
+  const player3 = useVideoPlayer(bigBuckBunnySource, (player) => {
+    player.loop = true;
+    player.play();
+  });
+  const player4 = useVideoPlayer(bigBuckBunnySource, (player) => {
+    player.loop = true;
+    player.play();
+  });
```

Comment:
```suggestion
	const initializePlayer = (player) => {
	  player.loop = true;
	  player.play();
	}
  const player1 = useVideoPlayer(bigBuckBunnySource, initializePlayer);
  const player2 = useVideoPlayer(bigBuckBunnySource, initializePlayer);
  const player3 = useVideoPlayer(bigBuckBunnySource, initializePlayer);
  const player4 = useVideoPlayer(bigBuckBunnySource, initializePlayer);
```
You need to add types to the `initializePlayer` lambda.